integration_tests/src/main/python/collection_ops_test.py

```diff
@@ -67,8 +67,8 @@ def test_concat_double_list_with_lit(dg):
 
 @pytest.mark.parametrize('data_gen', non_nested_array_gens, ids=idfn)
 def test_concat_list_with_lit(data_gen):
-    lit_col1 = f.lit(gen_scalar(data_gen)).cast(data_gen.data_type)
-    lit_col2 = f.lit(gen_scalar(data_gen)).cast(data_gen.data_type)
+    lit_col1 = f.lit(with_cpu_session(lambda spark: gen_scalar(data_gen))).cast(data_gen.data_type)
```

Comment:
This PR is intended to put `f.lit` into `with_cpu_session`, not only the `f.lit` in the `gen_scala` but also the `f.lit` in other places.   Maybe change to the following:
```
with_cpu_session(lambda spark: f.lit(gen_scalar(data_gen)).cast(data_gen.data_type))
```

Reply:
Please check `f.lit` in other places in this PR.

Reply:
Good catch, checked and fixed.